js_modules/dagster-ui/packages/ui-core/src/asset-graph/useAssetGraphSupplementaryData.oss.tsx

```diff
@@ -21,6 +21,9 @@ export const useAssetGraphSupplementaryData = (
   const loading = Object.keys(liveDataByNode).length !== nodes.length;
 
   const assetsByStatus = useMemo(() => {
+    if (loading) {
+      return emptyObject;
+    }
```

Comment:
This seems like a nice change but it's also pretty significant to not show any data until ALL the live data is ready? Just calling out to make sure this is intended!

Reply:
this is just for the status data. This means that you won't get any partial results from status filtering, instead you'll need to wait until all of the status data is loaded before status being taken into account. That said we do return a `loading: true` state so the UI doesn't even show partial results in the first place so basically this is just avoiding unnecessary work.